lib/active_record/tenanted/database_adapters/sqlite.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  module Tenanted
+    module DatabaseAdapters
+      class SQLite
+        def initialize(db_config)
+          @db_config = db_config
+          @configuration_hash = db_config.configuration_hash
+        end
+
+        def create_database
+          database_path = get_database_path(db_config)
+
+          # Ensure the directory exists
+          database_dir = File.dirname(database_path)
+          FileUtils.mkdir_p(database_dir) unless File.directory?(database_dir)
+
+          # Create the SQLite database file
+          FileUtils.touch(database_path)
+        end
+
+        def drop_database
+          database_path = get_database_path(db_config)
+
+          # Remove the SQLite database file and associated files
+          FileUtils.rm_f(database_path)
+          FileUtils.rm_f("#{database_path}-wal")  # Write-Ahead Logging file
+          FileUtils.rm_f("#{database_path}-shm")  # Shared Memory file
```

Comment:
This is the only potential breaking change but I think it's a bug fix? This adds also removing the `-wal `and `-shm` when we drop the database.